bootstrap.sh

```
@@ -1,20 +1,12 @@
 #!/bin/bash
-kubectl apply -f .infrastructure/mysql/ns.yml
-kubectl apply -f .infrastructure/mysql/configMap.yml
-kubectl apply -f .infrastructure/mysql/secret.yml
-kubectl apply -f .infrastructure/mysql/service.yml
-kubectl apply -f .infrastructure/mysql/statefulSet.yml
+kind delete cluster
```

Avoid deleting/affecting unexpected clusters and add safer shell options. Consider adding a safety/explicit cluster name or checking for the intended cluster before delete, and enable strict shell error handling (e.g., set -euo pipefail). This prevents silent failures and accidental deletion of other clusters. The current `kind delete cluster` without an explicit name may operate on the default cluster unexpectedly.